examples/liquids.lua

```diff
@@ -0,0 +1,120 @@
+register_option_bool("only_draw_at_mouse", "Testing: Draw only liquid num at mouse position", "", false)
```

Comment:
We usually add the `meta` information for the examples, since they are loaded into the overlunky automatically and shown in the scripts tab with the name and have small description on what they are

Reply:
oopsie I forgot